src/components/Drawer/Drawer.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+  <aside v-if="value" class="unnnic-drawer">
+    <section class="unnnic-drawer__overlay"></section>
+    <transition appear name="drawer">
+      <section
+        v-if="showDrawer"
+        :class="['unnnic-drawer__container', wide && 'unnnic-drawer__container--wide']"
+      >
+        <header class="unnnic-drawer__header">
+          <section class="unnnic-drawer__title-container">
+            <h1 class="unnnic-drawer__title unnnic-font secondary title-sm black">{{ title }}</h1>
```

Comment:
According to the frontend chapter wiki, it is not recommended to use common classes with BEM:

> Do not use common classes with BEM (such as class="menu__item large active"). In addition to removing the security of specificity, on certain occasions it would be difficult to differentiate between common classes and blocks (it should be class="menu__item menu__item--large menu__item--active").

Furthermore, this can confuse a developer when reading the "unnnic-drawer__title" styling and not seeing where the other stylings are coming from.